CHANGES.md

```diff
@@ -42,6 +42,7 @@ Features
 * [#422](https://github.com/twall/jna/pull/422): Update libffi to v3.2.1 - [@tdaitx](https://github.com/tdaitx).
 * [#425](https://github.com/twall/jna/pull/425): Add support for PPCLE - [@tdaitx](https://github.com/tdaix).
 * [#428](https://github.com/twall/jna/pull/428): Added Wincon.h related functions and definitions to `com.sun.jna.platform.win32.Kernel32` - [@lgoldstein](https://github.com/lgoldstein).
+* [#431](https://github.com/twall/jna/pull/431): Added named pipe API support [@lgoldstein](https://github.com/lgoldstein)
```

Comment:
It's missing a period at the end and  a dash before your name. Just trying to make all these look alike.

Furthermore, you didn't add named pipes API support, you added it for Win32. JNA supports many operating systems - this should be something much more specific (people *do* read this), like `Added named pipe support to `com.sun.jna.platform.win32.Kernel32`.